CONTRIBUTING.rst

```diff
@@ -269,21 +269,22 @@ Updating Testing Data
 
 If your code changes require changes to the testing data of `xclim` (i.e.: modifications to existing datasets or new datasets), these changes must be made via a Pull Request at the `xclim-testdata repository`_.
 
-`xclim` allows for developers to test specific branches/versions of `xclim-testdata` via the `XCLIM_TESTDATA_BRANCH` environment variable, either through export, e.g.::
+`xclim` allows for developers to test specific branches/versions or forks of the `xclim-testdata` repository via the `XCLIM_TESTDATA_BRANCH` and `XCLIM_TESTDATA_REPO` environment variables, respectively, either through export, e.g.::
```

Comment:
should we rename this repo at some point? if `xsdba` pulls from it too?

Reply:
We could. My changes here make it much easier to do in the future (though I'm running into trouble with other projects due to hard coding of values; working through it).

I wouldn't oppose doing a huge cleanup of the testing data.